proc_IR_noecho.py

```diff
@@ -115,6 +111,7 @@ def vec_lcurve(l):
     sfo1 = 272.05
     arbitrary_reference = s.get_prop('acq')['BF1'] 
     logger.info(strm("SFO1 is",sfo1))
+    s.ft('t2')
     s.setaxis('t2',lambda x:x + sfo1 - arbitrary_reference)
```

Comment:
Line 115 doesn't make sense unless in the frequency domain.
this has been commented on before, actually.

Reply:
OK, I see that it's already in the Fourier domain